demos/lp-vault-osmosis-upgrade/contract_setup.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/sh
+
+BINARY=quasarnoded
+HOME_QSR=$HOME/.quasarnode
+CHAIN_ID=quasar
+
+INIT1='{"lock_period":6,"pool_id":1,"pool_denom":"gamm/pool/1","base_denom":"uosmo","local_denom":"ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518","quote_denom":"stake","return_source_channel":"channel-0","transfer_channel":"channel-0","expected_connection":"connection-0"}'
+INIT2='{"lock_period":6,"pool_id":3,"pool_denom":"gamm/pool/3","base_denom":"uosmo","local_denom":"ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518","quote_denom":"fakestake","return_source_channel":"channel-0","transfer_channel":"channel-0","expected_connection":"connection-0"}'
+INIT3='{"lock_period":6,"pool_id":2,"pool_denom":"gamm/pool/2","base_denom":"uosmo","local_denom":"ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518","quote_denom":"usdc","return_source_channel":"channel-0","transfer_channel":"channel-0","expected_connection":"connection-0"}'
+
+cd ../../smart-contracts
+
+echo "Running store code"
+RES=$(quasarnoded tx wasm store artifacts/lp_strategy-aarch64.wasm --from alice --keyring-backend test -y --output json -b block --chain-id $CHAIN_ID --gas auto --fees 10000uqsr)
```

Comment:
Pls. use this in a variable lp_strategy-aarch64.wasm, for linux user :)

For linux users it should be set to lp_strategy.wasm and for mac users it should set to lp_strategy-aarch64.wasm

Reply:
Sure. I will put an if else condition in it.